sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1202,14 +1202,6 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
         });
         promises.push(deleteAssessmentPromise);
       }
-      if (application.applicationStatus === ApplicationStatus.Completed) {
-        const startAssessmentPromise =
-          this.workflowClientService.startApplicationAssessment(
-            application.workflowName,
-            application.currentAssessment.id,
-          );
-        promises.push(startAssessmentPromise);
```

Comment:
As these promises are not part of the async operation, do we still need the restrict the amount of parallel operations with respect to the cpu limit? Or does it need a refactor?

Reply:
The processes are already restricted by the amount of CPU cores. Line 1211 takes care only of some leftover (always lesser than the total CPU) cores.
Besides that, the only intention of this method right now is to trigger cancel operations, which will no longer be needed once the upcoming tickets for the queue consumers are done.